sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -138,6 +138,12 @@ export type EntityStatus = "Active" | "Creating" | "Deleting" | "ReceiveDisabled
 export interface GetMessageIteratorOptions extends OperationOptionsBase {
 }
 
+// @public
+export interface ISO8601Duration {
```

Comment:
I might have missed the conversation around this, but do we have to export this new interface? Do we plan to use it in more than one place? What other "kind" do we expect? If no other kind, then why should the user have to provide it?

Reply:
@xirzec asked for a new type to make it look cleaner. 😁

We might use it in more places in the future, mostly in any sort of "properties" like this one. That also reminds me of potentially adding the same to the SQL params. But I'll defer that for now.
There can be more kinds - "int" or "long" for example - if thinking only for interop.

Reply:
`kind` or `type` is a nice way for us to provide hints around ambiguous types (e.g. this string should be interpreted as a guid). 

Exporting the interface is really just a convenience for the consumer, since otherwise they have to type it out on their side or create their own, compatible interface. If we don't want to give it a name, we don't have technically have to.